scene/gui/graph_edit.cpp

```diff
@@ -610,7 +610,7 @@ void GraphEdit::_top_layer_input(const Ref<InputEvent> &p_ev) {
 									just_disconnected = true;
 
 									emit_signal(SNAME("disconnection_request"), E.from, E.from_port, E.to, E.to_port);
-									to = get_node(String(connecting_from)); //maybe it was erased
+									to = get_node({ connecting_from }); // Maybe it was erased.
```

Comment:
I'm not familiar with this syntax, what does it do exactly?

Reply:
I don't remember why I used this tbh. It's something similar to initializer list.

Reply:
`get_node` requires a `NodePath`, which only has a constructor with a `String` parameter. There is no implicit conversion from `StringName` directly to `NodePath`, since that would be a double implicit conversion, which is not allowed. So for example `get_node(NodePath(connecting_from))` would work, but either way it's basically the same conversion chain as before, so I would either change it to the explicit NodePath conversion like above since that's more readable IMO or leave these lines as they were.

Reply:
Ah right, I wanted to avoid StringName -> String -> NodePath conversion, so I was constructing NodePath here and found out about this syntax.

I can change it to NodePath() if that's better, I think it's the same.

Reply:
I think `{...}` is not really used accross the codebase hence I agree with @Geometror that it makes sense to go for either `NodePath(...)` or `String(...)`.